### PR TITLE
ROC-4470 additional AKS documentation

### PR DESCRIPTION
**Pull request by markry-kainos**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,33 @@ warning The case-insensitive file ..\cmc-citizen-frontend\node_modules\nyc\node_
...
+
+Get pods, service, ingress resources: `kubectl get po,svc,ing -n [namespace]`
+
+#### View Logs
+
+Get pod name: `kubectl get po -n [namespace]`
+
+View logs for pod: `kubectl logs [podname] -n [namespace]`
+
+*Note:* you can follow with `-f`
```
Comment (ian-mcwilliams) [inline README.md]: follow with? shouldn't the tail f go in the middle? eg

`kubectl logs -f [podname] -n [namespace]`

File: src/kubernetes/README.md [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
+- Install Kubernetes (k8s) cli: `az acs kubernetes install-cli`
 - Login: `az login`
 - Set azure subscription:  `az account set --subscription DCD-CNP-DEV`
 - Add to k8s cli (kubectl): `az aks get-credentials -n cnp-aks-cluster -g cnp-aks-rg`
+- Optional: install kubectx: https://github.com/ahmetb/kubectx ::: this helps switching namespace and contexts
 
 ## Post-Setup
 
 - Set context: `kubectl config use-context cnp-aks-cluster`
 - Test by showing k8s nodes: `kubectl get nodes`
```
Comment (ian-mcwilliams) [inline src/kubernetes/README.md]: ideally add something to identify what success looks like - "You should see..."

File: README.md [hunk 3]
```diff
@@ -119,6 +119,33 @@ warning The case-insensitive file ..\cmc-citizen-frontend\node_modules\nyc\node_
 warning The case-insensitive file ..\cmc-citizen-frontend\node_modules\nyc\node_modules\ansi-regex shouldn't be copied twice in one bulk copy
 ```
 
+## Preview Environment
+
+We get a fully functional environment in Azure Kubernetes (AKS) per pull request. For more
+info see: [K8S README](/src/kubernetes/README.md)
+
+Example URL using PR number from GitHub: `https://cmc-citizen-frontend-pr-XXX.service.core-compute-preview.internal/dashboard`
```
Comment (ian-mcwilliams) [inline README.md]: only for frontend - does a claimstore change have any way to rub on a frontend pulled from aat? it didn't in preview but would be good to clarify here either way

Comment (markry-kainos) [reply README.md]: I see what you mean - does this read better?

**Pull request merged**